### ISSUE TITLE
ACE cleanups for ACI/NH: different convergence behavior when running ACE problem w/o Erosion with Tempus vs. Piro

### ISSUE THREAD

**Issue by ikalash** [sandialabs/LCM#80]
In attempting to run an erosion-free MiniErosion denudation problem w/ both Piro and Tempus, I discovered that the convergence history is different.  Tempus requires much more nonlinear solves in the mechanical problem (see attached output).  I am conjecturing that the initial guess used in Tempus in each time-step is not the same as the one in Piro / not correct.  We may want to look into this if Tempus ends up being the method of choice for ACI/NH runs using ACE.  

Despite the different convergence history, Piro and Tempus produce the same solution for both the mechanical and the thermal problems.

NOTE: even though the Piro output file has "erosion" in it, the code was hacked to turn off Erosion.

[out_piro_erosion.txt](https://github.com/sandialabs/LCM/files/13253038/out_piro_erosion.txt)
[out_tempus_noerosion.txt](https://github.com/sandialabs/LCM/files/13253039/out_tempus_noerosion.txt)

**Comment by ikalash** [comment]
I forgot to mention: we should look at this after #78 has been resolved.

**Comment by lxmota** [comment]
Closing as won't fix — irrelevant after recent code changes.
